jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -4670,10 +4670,11 @@ Object setMethodHandleCache(Object cache) {
 	return result;
 }
 
-/*[IF Sidecar19-SE]*/
 ConstantPool getConstantPool() {
 	return new Access().getConstantPool(this);
```

Comment:
Since we are here, please use `SharedSecrets.getJavaLangAccess()` instead new an instance for every call.

Reply:
Sure, I will change that. Do we want to make the same change here?

Reply:
Or perhaps change `new Access().getConstantPool(this)` to call `getConstantPool()`

Reply:
`SharedSecrets.getJavaLangAccess()` saves one level method call though it is negligible after JIT. 
Another option is to change the method `Access.getConstantPool()` to `static`, but I will let that to you.

Reply:
Pushed the changes.

Reply:
`SharedSecrets` is in `jdk.internal.access` for JDK9+, `sun.misc` prior to that. Will add in both imports ifdeffed on the Java version.